### PR TITLE
Fixed Compile errors addresses #140 and #141

### PR DESCRIPTION
**Pull request by SamanthaDawsonBanks**
Minor compile errors

### CODE REVIEW

File: mcomp_C/movement/Propulsion.cpp [hunk 1]
```diff
@@ -31,10 +31,6 @@ Propulsion::Propulsion() {
 
 Propulsion::~Propulsion() {
   releaseMotors();
-  delete leftMotor;
```
Comment (SamanthaDawsonBanks) [inline mcomp_C/movement/Propulsion.cpp:34]: there are not new so no need to delete

File: mcomp_C/movement/Propulsion.cpp [hunk 2]
```diff
@@ -138,6 +134,6 @@ void Propulsion::releaseMotors(){
 
 
 double Propulsion::getHeading(){
-  return currentHeading();
```
Comment (SamanthaDawsonBanks) [inline mcomp_C/movement/Propulsion.cpp:141]: Typo

File: mcomp_C/movement/Propulsion.h [hunk 3]
```diff
@@ -3,9 +3,8 @@
  *
  *  Created on: 26 Mar 2018
  *      Author: Stephen Pope 15836791
-
  *      Author: David Avery 15823926
-
```
Comment (SamanthaDawsonBanks) [inline mcomp_C/movement/Propulsion.h:8]: missed end of line

File: mcomp_C/tests/Propulsion_test.cpp [hunk 4]
```diff
@@ -5,7 +5,7 @@
  *      Author: Stephen Pope 15836791
  */
 
-#include "Propulsion.h"
```
Comment (SamanthaDawsonBanks) [inline mcomp_C/tests/Propulsion_test.cpp:8]: bad path

**Review by stevempope — APPROVED** [review]

**Review by HarryWJackson1996 — APPROVED** [review]

**Pull request merged**